source/release_notes/4.2.8-8.rst

```diff
@@ -0,0 +1,22 @@
+.. _PSMDB-4.2.8-8:
+
+================================================================================
+*Percona Server for MongoDB* 4.2.8-8
+================================================================================
+
+:Date: July 7, 2020
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/4.2/install/index.html>`_
+
+Percona Server for MongoDB 4.2.8-8 is an enhanced, open source, and highly-scalable database that is a
+fully-compatible, drop-in replacement for `MongoDB 4.2.8 Community Edition <https://docs.mongodb.com/manual/release-notes/4.2/#june-15-2020>`_.
+It supports MongoDB 4.2.8 protocols and drivers.
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PSMDB-630`: Some LDAP servers drop idle connections upon timeout. Both this ticket and the :jirabug:`PSMDB-583` are fixed by a single patch. (Thanks to user mdiorio for reporting this issue)
+* :jirabug:`PSMDB-656`: LDAP - user's permissions remain intact after a user is removed from LDAP
```

Comment:
This bug was only during development, it has never affected users because it was never released.
So we shouldn't list this in the release notes.
```suggestion
```